Assets/PatchKit Patcher/Resources.meta

```diff
@@ -1,9 +1,8 @@
 fileFormatVersion: 2
 guid: bfabb18354a957c48b1eddc152e3d4fe
 folderAsset: yes
-timeCreated: 1465833977
-licenseType: Free
 DefaultImporter:
+  externalObjects: {}
```

Comment:
These changes can be discarded, as they are a result of import to newer Unity.

Reply:
done